### ISSUE TITLE
How to remove one of my native listener instead of removing all of them by using Interceptor.detachAll()?

### ISSUE THREAD

**Issue by abcdefgnihao** [frida/frida#1698]

**Comment by iGio90** [comment]
var x = Interceptor.attach(...);

x.detach();

**Comment by abcdefgnihao** [comment]
Ty
